staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -402,10 +402,10 @@ func (rt *debuggingRoundTripper) RoundTrip(req *http.Request) (*http.Response, e
 	reqInfo.complete(response, err)
 
 	if rt.levels[debugURLTiming] {
-		klog.Infof("%s %s %s in %d milliseconds", reqInfo.RequestVerb, reqInfo.RequestURL, reqInfo.ResponseStatus, reqInfo.Duration.Nanoseconds()/int64(time.Millisecond))
+		klog.Infof("%s %s %d in %d milliseconds", reqInfo.RequestVerb, reqInfo.RequestURL, reqInfo.ResponseStatusCode, reqInfo.Duration.Nanoseconds()/int64(time.Millisecond))
```

Comment:
this breaks [this (admittedly fragile) test](https://github.com/kubernetes/kubernetes/blob/master/test/cmd/get.sh#L108-L119)